contracts/Remittance.sol

```diff
@@ -19,4 +116,8 @@ contract Remittance is Ownable, Pausable {
     function unpause() public onlyOwner {
         super._unpause();
     }
+
+    function hash(string memory password) public pure returns(bytes32) {
+        return keccak256(abi.encodePacked(password));
+    }
```

Comment:
Now you have 2 ways to hash the password. The way found in this function and the way found on line 82.
I hope you keep them forever in sync, because if you forget to keep them in sync just once, then you advertise one way that should work, but it actually fails when it is time to release the funds...